d2l-image.js

```diff
@@ -1,11 +1,27 @@
-import '@polymer/polymer/polymer-legacy.js';
-import '@polymer/iron-ajax/iron-ajax.js';
-import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
-const $_documentContainer = document.createElement('template');
+import 'd2l-fetch/d2l-fetch.js';
+import { css, html, LitElement, nothing } from 'lit';
+import { auth } from 'd2l-fetch-auth/d2l-fetch-auth.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
 
-$_documentContainer.innerHTML = `<dom-module id="d2l-image">
-	<template strip-whitespace="">
-		<style>
+class D2LImage extends LitElement {
+	static get properties() {
+		return {
+			/**
+			 * Alternate text describing the image
+			 * @type {string}
+			 */
+			altText: { type: String, attribute: 'alt-text' },
```

Comment:
This change in attribute name (and the token change) will be a breaking change for consumers. I see you have that in the title of the PR, although _technically_ I don't think migrating something to Lit necessarily means it needs to be a breaking change (assuming the API stays the same).